specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -104,14 +104,31 @@ function TreeView<SCHEMA extends AnyTree>({
   );
 
   // FEATURE: synchronize focus path with the URL
-  const [focusPath = [], setFocusPath] = useCachedState(
+  const [_focusedPath = [], setFocusPath] = useCachedState(
     'tree',
     `focusPath${tableName}`
   );
+  const [focusPath1 = [], setFocusPath1] = React.useState<RA<number>>();
```

Comment:
(optional) create a hook that will return all the states you need in a single object, and then just call that hook twice.

i.e:
```ts
const leftTreeStates = useTreeStates();
const rightTreeStates = useTreeStates();
```

or even:
```ts
const treeStates = {
  left: useTreeStates(),
  right: useTreeStates(),
};

const [lastFocusedTree, setLastFocusedTree] = React.useState<'left' | 'right'>('left');
const state = treeStates[lastFocusedTree];
```

now treeStates.left always stores the states specific to the left tree, treeStates.right always stores the states specific to the right tree, and state always denotes the currently focused tree